build/litsearch/getInternalData.py

```diff
@@ -212,6 +212,27 @@ def getRelevantPRData():
     return textForReviewPRs
 
 
+# Get Mt. Sinai data
+def addMtSinaiReviewLinks(df):
+    # Get list of papers reviewed
+    mtSinaiPapersResponse = requests.get("https://api.github.com/repos/ismms-himc/covid-19_sinai_reviews/contents/markdown_files", headers=headers)
```

Comment:
Because this is pulling directly from the latest version of the `covid-19_sinai_reviews` and our appendix is updated manually, can we also use this to flag when we should update the appendix contents?

Reply:
I added in a check for this and have it logging out when there are new reviews not in our paper (there are currently 25 new reviews not in our paper). 

You could consider automating this (automatically update the appendix when new reviews are available) or adding in some other form of notification (like opening a new issue via the api or something) but hopefully being able to see it in the logs is helpful.